internal/dnfjson/cache_test.go

```diff
@@ -329,3 +321,64 @@ func TestDNFCacheCleanup(t *testing.T) {
 	_, ok := cache.Get("notreallyahash")
 	assert.False(t, ok)
 }
+
+func TestCleanupOldCacheDirs(t *testing.T) {
+	// Run the cleanup without the cache present and with dummy distro names
+	CleanupOldCacheDirs("/var/tmp/test-no-cache-rpmmd/", []string{"fedora-37", "fedora-38"})
```

Comment:
To avoid any issues with deleting files that aren't ours (unlikely, but stranger things have happened), maybe create another `t.TempDir()` and run this against a subdirectory of that new dir.

Reply:
Not sure what you mean. This is testing to make sure there is no panic when the directory doesn't exist.

Reply:
It's not important though.